### PR TITLE
Add instructions on ARM64 target build for Json Generation Layer

### PR DESCRIPTION
**Pull request by kaizhangNV**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -231,6 +231,50 @@ To use, simply push it to the device and run it.  The resulting json file will b
...
+# Setup resolv.conf for DNS server
+sudo -E mv ${TARGET_ROOFS}/etc/resolv.conf ${TARGET_ROOFS}/etc/resolv.conf.saved
+sudo -E cp /etc/resolv.conf ${TARGET_ROOFS}/etc
+
+# Run chroot to target system sandbox
+sudo LC_ALL=C chroot . /bin/bash
+```
+#### Build on Target System
+- Run the following instructions on the Target machine.
+    - Install the dependencies following [Ubuntu System Requirements](BUILD.md#ubuntu-system-requirements).
```
Comment (dgkoch) [inline BUILD.md]: The build instructions in this file seem to be split up into (a) System requirements, and (b) actually building.  It seems like most/all of the steps above this comment should be up in a System Requirements sub-section.

**Review by dgkoch — CHANGES_REQUESTED** [review]

File: BUILD.md (hunk 1 above)
Comment (kaizhangNV) [reply BUILD.md]: I reorganized the instructions.

File: BUILD.md [hunk 2]
```diff
@@ -176,7 +206,8 @@ ctest -C Debug --output-on-failure --parallel 16
 ctest -C Release  --output-on-failure --parallel 16
 ```
 
-### Linux and macOS Build
+### Linux AMD64, Linux ARM64 and macOS Build
+For Linux ARM64 build, run the following commands in [chroot sandbox environment](BUILD.md#linux-arm64-additional-system-requirements).
```
Comment (dgkoch) [inline BUILD.md]: This makes if seem like the commands below are are only for ARM64 and there are no instructions for AMD64/MacOS.

Suggest moving this sentence below the code block and change it to: 

```suggestion
For Linux ARM64 build, run the above commands in the [chroot sandbox environment](BUILD.md#linux-arm64-additional-system-requirements).
```

File: BUILD.md [hunk 3]
```diff
@@ -232,6 +263,7 @@ To use, simply push it to the device and run it.  The resulting json file will b
 /sdcard/Android/<output>.json
 ```
 
+
```
Comment (dgkoch) [inline BUILD.md]: extraneous change

**Review by dgkoch — APPROVED** [review]
LGTM

**Pull request merged**